include/RAJA/exec-cuda/reduce_cuda.hxx

```diff
@@ -1032,7 +1037,7 @@ class ReduceMinLoc<cuda_reduce<BLOCK_SIZE>, T> {
           RAJA_MINLOC(sd[threadId],
                       m_blockdata[m_blockoffset + blockId + 1]);
       __threadfence();
-      int oldBlockCount = atomicAdd(&retiredBlocks[m_myID], (int)1);
+      unsigned int oldBlockCount = ::atomicAdd((unsigned int*)&retiredBlocks[m_myID], (unsigned int)1);
```

Comment:
Holger, what about for the older versions of NVCC?  Don't we still need our implementation of atomicAdd for those?